docs/source/markdown/podman-rm.1.md

```diff
@@ -56,6 +60,11 @@ Remove a container by its name *mywebserver*
 $ podman rm mywebserver
 ```
 
+Remove a *mywebserver* container and all of the containers that depend on it
+```
+$ podman rm --depend mywebserver
```

Comment:
Does this print every container that was removed? I got a little concerned because there are no changes to cmd/podman

Reply:
Yes, the container is added to the reports list when being removed in the API.  The Podman client just walks the list.

Reply:
Added a test to verify.